app/ports/templates/ports/about.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'ports/layout.html' %}
+
+{% block title %}About | {% endblock %}
+{% block content %}
+    <br>
+    <h3>About</h3>
+    <hr>
+    <div class="row">
+        <div class="col-lg-3 col-md-4 bg-light border border-secondary p-2">
+            <strong>Quick Links</strong>
+            <hr>
+                <ul>
+                    <li><a href="{% url 'stats_faq' %}">Statistics FAQ</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp">Git Repository</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp/tree/master/docs">Documentation</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp/tree/master/docs/api_v1">API</a></li>
+                    <li><a href="https://docs.google.com/document/d/1TqNHm3NrtLDelWqY7P4pZyTMZDJrlsPymFZklzV2t24/edit">Feature Requests</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp/issues">Issues</a></li>
+                    <li><a href="https://www.macports.org/install.php">Install MacPorts</a></li>
+                </ul>
+        </div>
+        <div class="col-lg-9 col-lg-8">
+            The MacPorts web application:
+            <ol>
+                <li>Displays general information about all the ports.</li>
+                <li>Fetches and displays build history of the ports.</li>
+                <li>Collects and displays installation statistics (system configurations and installed ports) from users
+                    who opt-in by installing the port mpstats.
+                </li>
+                <li>Makes available all the collected data through API.</li>
+            </ol>
+            <p>The application is based on the <a href="https://www.djangoproject.com">Django Framework</a> and runs
```

Comment:
Maybe the list of technologies could be moved to the bottom of credits?
Technologies being used certainly deserve to be credited, but with a much lower priority than the author of the app.

Then you could also avoid repeating the author and just mention the author once, at the very top of credits, as it is now.